rules/optimise-js-with-lodash/rule.md

```diff
@@ -0,0 +1,84 @@
+---
+type: rule
+title: Do you optimise your JavaScript code with Lodash wisely and efficiently?
+uri: optimise-js-with-lodash
+authors:
+  - title: Jack Pettit
+    url: https://www.ssw.com.au/people/jack-pettit/
+created: 2023-08-01T23:51:13.657Z
+guid: 14157333-05fd-492d-a443-6eeb389a654a
+---
+Do you want to make your JavaScript code more efficient and easier to read? Lodash might be the utility library you need. But like any tool, knowing when and how to use it is crucial for the maintainability and performance of your project.
+
+JavaScript is a powerful language, but it can sometimes be tricky to write clean, efficient code. This is where Lodash shines. Lodash is a JavaScript utility library providing handy methods for manipulating and combining arrays, objects, numbers, strings, among other data types.
+
+<!--endintro-->
+
+::: info  
+For optimal bundle size and tree-shaking capabilities in modern JavaScript applications, we recommend using [lodash-es](https://www.npmjs.com/package/lodash-es) over the standard lodash library.
+:::
```

Comment:
cc @christoment 

can you do tree shaking with standard one?
This one looks a bit outdated maybe